arbitrary-file-write/file-create/README.md

```diff
@@ -49,4 +49,21 @@ asdas
 
 [we]! can have symbols and garbage at the end
 <00><ff> binary also ok here
-```
\ No newline at end of file
+```
+### Shell Profiles
+#### Login Shells
+`--login` flag. 
+* `/etc/profile` executed for all users
+* First found of `~/.bash_profile`, `~/.bash_login`, and `~/.profile` is then executed
+#### Non-login shell
+* `/etc/bash.bashrc` executed for all users 
+* `~/.bashrc` for user specific.  
+
+### Eicar antimalware testfile
```

Comment:
Generally I don't think it's really useful to an attacker. It's a funny "trick" to trip up antivirus, but it doesn't help you to elevate priviledges.

HOWEVER, this got me thinking. We could investigate if there are antivir that by default delete a file. and maybe this could be used to force delete log file etc. This would be useful to an attacker. But definitely requires more investigation and documentation which antivir this affects.